templates/gc-audience-en.html

```diff
@@ -12,7 +12,7 @@
 <div class="row profile">
 	<div class="col-md-6">
 		<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras sagittis massa sed libero eleifend, ac iaculis urna tempor. Cras sagittis massa sed libero eleifend, ac iaculis urna tempor.</p>
-		{{>follow}}
+		{% include gc-follow-us/gc-follow-us.html isInline=true %}
```

Comment:
Should we use include relative?

Reply:
Unfortunately, `include_relative` does not allow the use of "../" to go up a folder. Furthermore, `include_relative` seems to only be available inside the `_posts` folder.